READ_BEFORE_UPDATE.md

```diff
@@ -16,13 +16,18 @@
 * docs: start fixing incorrect camel case by @fritterhoff in https://github.com/eduMFA/eduMFA/pull/25
 * docs: cleanup old refs and fix path names by @j-hoff in https://github.com/eduMFA/eduMFA/pull/28
 * docs: use fu repo by @fritterhoff in https://github.com/eduMFA/eduMFA/pull/33
-* docs: add docu for new sms provider "Http Message To Uid Provider" by @j-hoff in https://github.com/eduMFA/eduMFA/pull/21
-* feat: smstoken: add configurable URL which is called after check, extend test by @j-hoff in https://github.com/eduMFA/eduMFA/pull/18
-* feat: make result value available in logginghandler, verbosely log set tokeninfo by @j-hoff in https://github.com/eduMFA/eduMFA/pull/19
-* feat: new API endpoint POST /info/<serial> to bulk modify tokeninfo by @j-hoff in https://github.com/eduMFA/eduMFA/pull/20
+* docs: add docu for new sms provider "Http Message To Uid Provider" by @j-hoff
+  in https://github.com/eduMFA/eduMFA/pull/21
```

Comment:
Is there a reason for the new line breaks? (Some formatting anywhere that requires a specific line length? 🤔 )

Reply:
This is the automated code format from Jetbrain IDEs